File: vpon_plugin_poc/android/src/main/kotlin/io/flutter/plugins/vponmobileads/VponFlutterPlatformView.java
package io.flutter.plugins.vponmobileads;

import android.view.View;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import io.flutter.plugin.platform.PlatformView;

class VponFlutterPlatformView implements PlatformView {

    @Nullable
    private View view;

    VponFlutterPlatformView(@NonNull View view) {
        this.view = view;
    }

    @Nullable
    @Override
    public View getView() {
        return view;
    }

    @Override
    public void dispose() {
        this.view = null;
    }

}

File: vpon_mobile_ads/android/src/main/kotlin/io/flutter/plugins/vponmobileads/VponFlutterPlatformView.kt
package io.flutter.plugins.vponmobileads

import android.view.View
import io.flutter.plugin.platform.PlatformView

internal class VponFlutterPlatformView(private var view: View?) : PlatformView {
    override fun getView(): View? {
        return view
    }

    override fun dispose() {
        view = null
    }
}
